tests/snuba/api/endpoints/test_organization_sessions.py

```diff
@@ -1085,6 +1087,147 @@ def test_mix_known_and_unknown_strings(self):
         )
         assert response.status_code == 200, response.data
 
+    @freeze_time(MOCK_DATETIME)
+    def test_release_semver_filter(self):
+        r1 = self.create_release(version="ahmed@1.0.0")
+        r2 = self.create_release(version="ahmed@1.1.0")
+        r3 = self.create_release(version="ahmed@2.0.0")
+
+        for r in (r1, r2, r3):
```

Comment:
wdyt to refactor it:
```
for release_name in ("ahmed@1.0.0", "ahmed@1.1.0", "ahmed@2.0.0"):
    release = self.create_release(version="ahmed@1.0.0")
    self.store_session(make_session(self.project, release=release.version))
```

Reply:
also we can you please apply it to other similar tests?

Reply:
Will handle in a subsequent PR